src/shared/hooks/useTimeAgo.test.ts

```diff
@@ -0,0 +1,26 @@
+import { add } from 'date-fns';
+import { renderHook } from '@testing-library/react';
+
+import { useTimeAgo } from './useTimeAgo';
+
+jest.mock('react-i18next', () => ({
+  useTranslation: () => ({
+    i18n: {
+      changeLanguage: () => new Promise(() => {}),
+      language: 'en',
+    },
+  }),
+  initReactI18next: {
+    type: '3rdParty',
+    init: () => {},
+  },
+}));
+
+describe('useTimeAgo hook tests', () => {
+  test('useTimeAgo should format date', () => {
```

Comment:
maybe also try some false value, incorrect data, future data, and minutes ago and just now?